app/bundles/PageBundle/Tests/Controller/PreviewFunctionalTest.php

```diff
@@ -4,25 +4,46 @@
 
 namespace Mautic\PageBundle\Tests\Controller;
 
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+use Doctrine\Persistence\Mapping\MappingException;
+use Mautic\CoreBundle\Security\Permissions\CorePermissions;
 use Mautic\CoreBundle\Test\MauticMysqlTestCase;
 use Mautic\DynamicContentBundle\Entity\DynamicContent;
 use Mautic\LeadBundle\Entity\Lead;
 use Mautic\PageBundle\Entity\Page;
+use Mautic\UserBundle\Entity\User;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
 class PreviewFunctionalTest extends MauticMysqlTestCase
 {
-    public function testPreviewPage(): void
+    /**
+     * @throws MappingException
+     * @throws ORMException
+     * @throws OptimisticLockException
+     */
```

Comment:
I don't think we need these comments for test methods. We don't do that in any other tests, just in this file. The test methods aren't used in other places just called automatically by PHPUNIT. Let's remove those here and other methods in this file.